recipe/run_conda_forge_build_setup_linux

```diff
@@ -52,6 +52,13 @@ if [[ ! -z "$CUDA_HOME" && -d /usr/local/cuda-9.2 ]]; then
   if [[ ! -f "/usr/local/cuda-10.0/compat/libcuda.so" ]]; then exit 1; fi
 fi
 
+# Export CONDA_OVERRIDE_CUDA to allow __cuda to be detected on CI systems without GPUs
+CUDA_VERSION="$(cat ${CI_SUPPORT}/${CONFIG}.yml | shyaml get-value cuda_compiler_version None)"
+if [[ "$CUDA_VERSION" == "None" ]]; then
+    CUDA_VERSION=""
+fi
+echo "export CONDA_OVERRIDE_CUDA='${CUDA_VERSION}'" > "${CONDA_PREFIX}/etc/conda/activate.d/conda-forge-ci-setup-activate.sh"
```

Comment:
Can you move this inside the if block?

Reply:
You also need to actually export this in this script for some CI

Reply:
I moved it out of the "if" block earlier because I thought the "if" above only deals with a CUDA 9.2 workaround. It has `-d /usr/local/cuda-9.2` as a condition. Can you confirm whether that's correct? I'll add the export.

Reply:
No, `$CUDA_VERSION=None` is for CPU only builds, so in that case `CONDA_OVERRIDE_CUDA` should not be set at all.